src/packages/calendaritem/calendaritem.taro.tsx

```
@@ -279,87 +264,73 @@ export const CalendarItem = React.forwardRef<
+          if (compareDate(propEndDate, currentDate[1])) {
+            currentDate[1] = propEndDate
+          }
+          defaultData = [
+            ...splitDate(currentDate[0]),
+            ...splitDate(currentDate[1]),
+          ]
+          break
+        case 'multiple':
+          // eslint-disable-next-line no-case-declarations
+          const defaultArr = [] as string[]
+          // eslint-disable-next-line no-case-declarations
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)